pages/listings/[id].tsx

```diff
@@ -328,6 +411,21 @@ function ListingPage(
               paddingTop={["0.5in", "0.5in", "0.5in", "1in"]}
               paddingBottom={[0, 0, 0, "1in"]}
             >
+              {isVisibilityToggled === "hidden" && (
+                <Card background="pink" width="100%" padding="2rem 2rem">
+                  <Stack>
+                    <Heading6 textAlign="center">
+                      This listing is hidden and cannot be publicly viewed.
+                    </Heading6>
+                    <ButtonPrimary
+                      onClick={() => onVisibilityUpdate(listing?.id)}
+                    >
+                      Change to public listing
+                    </ButtonPrimary>
+                  </Stack>
+                </Card>
+              )}
+              {isVisibilityToggled === "hidden" && <br />}
```

Comment:
Nitpick: instead of having `isVisibilityToggled === "hidden"` repeated twice, can you either add some `marginBottom` to the `Card` above or include the `<br />` above also?

Personally I'd prefer the 1st option, but if you want to do the 2nd, then you'd wrap it in a React fragment or empty tag to avoid returning multiple elements. Something like this structure:

```jsx
<> // <-- notice this
  <Card>
    ...
  </Card>
  <br />
</>
```